### PR TITLE
[New] Media Player Component

### LINKED ISSUE #55: Make multimedia player

### PR DESCRIPTION
**Pull request by YaBoiBreezy**
⭐ **New Features:**

- description of feature 1
- description of feature 2

ℹ️ **Related Issues:**

resolves #55 

📷 **Screenshots:** _(if applicable)_

### CODE REVIEW

File: src/components/MultimediaPlayer/stories/MultimediaPlayer.figCaption.stories.mdx
Comment (Rizvida1) [inline src/components/MultimediaPlayer/stories/MultimediaPlayer.figCaption.stories.mdx:16]: Make sure the capitalization is correct

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

**Comment by YaBoiBreezy** [comment]
Need shareWidget to finish this, so can't merge until shareWidget is merged

**Review by Rizvida1 — APPROVED** [review]

**Review by Rizvida1 — APPROVED** [review]

**Pull request merged**
